### PR TITLE
feat: product page

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added product page
- added footer in login and signup pages
- refactored CSS and added base classes needed in e-commerce

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me and the preview webpage is also looking clean and well spaced.
Good work on making the base classes for reducing the work on the other files!!

File: stylesheets/product.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.card-container {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    gap: 1rem;
+}   
+
+.card {
+    width: 18rem;
+    /* background-color: var(--lynx-grey-color); */
```
Comment (ShreyashMahajan) [inline stylesheets/product.css]: unwanted comments can be removed

**Review by ShreyashMahajan — APPROVED** [review]
UI as well as the Crypto store concept is very good. Good to merge

File: stylesheets/product.css (hunk 1 above)
Comment (harshitadaswani) [reply stylesheets/product.css]: done!

Comment (tushartiwari7) [inline stylesheets/product.css]: please remove comments

**Review by tushartiwari7 — APPROVED** [review]
Code is very consistent. You can merge this.

**Pull request merged**